Model/src/GamePackage/Die.java

```diff
@@ -0,0 +1,53 @@
+package GamePackage;
+
+import java.util.Random;
+
+/*****************************************************************
+ The Die Class will be a random number generator that "rolls"
+ a die some methods to return the value of the die.
+
+ @author Dylan Kernohan
+ @version Winter 2018

```

Comment:
According to the GVSU coding style the @version should have either a number or the date of creation. add the full date.

Reply:
are we doing date or version number??